tests/rust-integration-tests/integration_test/src/tests/seccomp_notify/seccomp_agent.rs

```diff
@@ -47,14 +50,12 @@ pub fn recv_seccomp_listener(seccomp_listener: &Path) -> SeccompAgentResult {
         Ok(msg) => msg,
         Err(e) => {
             let _ = unistd::close(conn);
-            let _ = unistd::close(socket);
```

Comment:
We can remove `unisted::close` because `OwnedFd` is automatically closed by `Drop`, can't we?

Reply:
Is there a way to check if `fd` is leaking?